### PR TITLE
Workflow updates

### PR DESCRIPTION
**Pull request by wdower**
- Bumps up versions for github action workflows
- Bumps the version of node used by the workflows to 16
- fixes a busted test for the backend

### CODE REVIEW

**Review by Amndeep7 — CHANGES_REQUESTED** [review]

File: .github/workflows/backend-tests.yml [hunk 1]
```diff
@@ -34,9 +34,9 @@ jobs:
            ${{ runner.os }}-
 
      - name: Setup Node.js
-       uses: actions/setup-node@v1
+       uses: actions/setup-node@v3
```
[the remark this answers is not in the document]
Comment (em-c-rod) [reply .github/workflows/backend-tests.yml:37]: I did remove the actions/cache action when updating workflows in `ts-inspec-objects`

**Review by em-c-rod — CHANGES_REQUESTED** [review]
Thanks for updating these workflows. I think we can remove the actions/cache actions and use the actions/setup-node to cache dependencies.

Ex:
```
- uses: actions/setup-node@v3
  with:
    node-version: 16
    cache: 'yarn'
    cache-dependency-path: subdir/package-lock.json 
 ```
 
More information about the cache-dependency-path can be found here: https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#caching-packages-data
I am not sure if we can utilize the cache-dependency-path without storing the library lock files.

File: .github/workflows/auto-approve-and-merge.yml [hunk 2]
```diff
@@ -11,8 +11,10 @@ jobs:
     name: Auto-approve dependabot PRs
     if: github.event.pull_request.user.login == 'dependabot[bot]' && contains(github.event.pull_request.labels.*.name, 'dependencies')
     runs-on: ubuntu-20.04
+    permissions:
+      pull-requests: write
```
Comment (Hookwitz) [inline .github/workflows/auto-approve-and-merge.yml:15]: I believe the addition of `contents: write` is required for permissions. I have been attempting to enable a similar workflow in the jsonix repo and had to add this permission to actually merge.

**Review by Hookwitz — CHANGES_REQUESTED** [review]

File: setup-dev-env.sh [hunk 3]
```diff
@@ -1,47 +1,47 @@
 #!/bin/bash
 
-if [ -f apps/backend/.env ]; then
-	echo "apps/backend/.env already exists, so all unset environment variables will now be filled by default values. If you would like to completely regenerate your secrets, please delete this file and then re-run the script."
+if [ -f .env ]; then
```
[the remark this answers is not in the document]
Comment (wdower) [reply setup-dev-env.sh:3]: Good catch.

**Review by em-c-rod — APPROVED** [review]

**Review by Hookwitz — APPROVED** [review]

**Pull request merged**